src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -52,11 +55,22 @@ public void Abort()
          {
             View.ValidationIsRunning(true);
             await _batchStarterTask.StartBatch(_outputFolderDTO.FolderPath, _cancellationTokenSource.Token);
```

Comment:
Also we should add some message to the log
Something like: Starting batch calculation

Then Starting comparison... etc